csatar-plugins/csatar/controllers/JsonImport.php

```diff
@@ -1125,6 +1129,38 @@ public function onProcessRegistrationForms() {
         }
     }
 
+    public function onProcessMemberCardsData() {
+        $dataFile = Input::file('member_cards_data_file');
+
+        if ($dataFile->isValid()) {
+            $dataFile = $dataFile->move(temp_path(), $dataFile->getClientOriginalName());
+            $memberCardsData = collect(json_decode(file_get_contents($dataFile->getRealPath())));
+        }
+
+        foreach ($memberCardsData as $memberCardData) {
+            $data = $memberCardData->fields;
+            $ecset_code = $data->tag[0];
+            $scout = Scout::withTrashed()->where('ecset_code', $ecset_code)->first();
+
+            if (empty($scout)) {
+                Log::warning("Could not find scout with identifier: $ecset_code. Member card data from not imported.");
+                continue;
+            }
+
+            $membershipCard = MembershipCard::firstOrNew(['rfid_tag' => $data->rfid_tag ]);
+
+            $membershipCard->scout_id           = $scout->id;
+            $membershipCard->issued_date_time   = $data->legyartva;
+            $membershipCard->active             = $data->ervenyes;
+            $membershipCard->note               = $data->megjegyzes;
```

Comment:
Ezek nem kene angolul legyenek?

Reply:
Nem, itt ami magyarul van, az a json-be is magyarul van.